AwsCryptographyPrimitives/runtimes/python/src/aws_cryptography_primitives/smithygenerated/aws_cryptography_primitives/plugin.py

```diff
@@ -0,0 +1,36 @@
+# Code generated by smithy-python-codegen DO NOT EDIT.
+
+from .config import Config, Plugin, smithy_config_to_dafny_config, CryptoConfig
+from smithy_python.interfaces.retries import RetryStrategy
+from smithy_python.exceptions import SmithyRetryException
+from .dafnyImplInterface import DafnyImplInterface
+
+def set_config_impl(config: Config):
+    '''
```

Comment:
Is it intentional for these to not be [PEP-0257 compliant docstrings](https://peps.python.org/pep-0257/)?

Reply:
Nope, will change